completed gsp implementation

ShwethaSureshKumar opened the pull request:
GSP algorithm on sequential data has been implemented.

therapist3003 reviewed (changes requested):
1. Make sure you keep all your datasets within /Datasets directory
2. If you're using any third-party modules, not mentioned in requirements.txt, make sure you add them to it. (prettytable)

ShwethaSureshKumar commented:
Moved the dataset files to Dataset/PrescriptiveAnalaysis1/GSP folder. Added requirements for prettytable in requirements.txt.
In addition, src/PrescriptiveAnalaysis1Apriori-Graph folder has also been added and its corresponding datasets have been uploaded in the Dataset/PrescriptiveAnalaysis1 folder.

therapist3003 reviewed (approved):
GSP algorithm implemented & refactored location of sample datasets
